kwanyong/app.js

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();     
+
+const express = require('express');
+const logger = require('morgan');
+const cors = require('cors');
+const { DataSource } = require('typeorm');
+
+
+const appDataSource = new DataSource({
+    type: process.env.DB_CONNECTION,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE
+})
+
+appDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+
+const app = express()
+ 
+app.use(cors());                  
+app.use(logger('combined'));
+app.use(express.json());
+
+const port = process.env.PORT
```

Comment:
해당 코드는 `app.listen()` 함수 내부에서 서버를 실행할 port 변수를 선언하고 할당하는 코드입니다.

app.listen() 함수에서만 사용하기 때문에 해당 함수와 함께 작성해준다면 더 좋을 것 같습니다!

```suggestion
// 예시
const port = process.env.PORT
app.listen(port, () => { ... }):
```